test/python/picongpu/compiling/distribution.py

```diff
@@ -46,16 +46,16 @@ def test_uniform(self):
         uniform_dist = picmi.UniformDistribution(density=8e24)
         self._compile_distribution(uniform_dist)
 
-    def test_analytic(self):
-        analytic_dist = picmi.AnalyticDistribution(
-            density_expression="a * sin(x) + z - sqrt(y)",
-            a=42)
-        self._compile_distribution(analytic_dist)
+#    def test_analytic(self):
```

Comment:
Why is this test removed?
We should delete the dead code because dead code (commented-out code) stays typically for ever. If you like to reintroduce the code part later please add a todo with a permanent link to the current code section in the development branch.

Reply:
Valid Point,

I commented it out because we currently do not support the analytic or gaussian-bunch distributions.

Both are planned to be implemented in the future, the gaussian bunch one in January, the analytic one will require some conceptual work. Thomas plans to set a coming student to it soon.

Will create an issue to remind me to either remove the code or finish the implementation.

Reply:
Done